about.html

```diff
@@ -0,0 +1,200 @@
+      >
+        <h2 class="text-center m-0 fw-bold">
+          AI Brains<br />Global Summit 2023 Logo
+        </h2>
+        <div class="aboutguide-bar m-0 d-flex"></div>
+        <small class="w-80"
+          >The logo of AI-B Global Summit 2023 was decided through the logo
+          competition from 13.December to 14 February.</small
+        >
+        <div
+          class="logo-container container my-4 d-flex justify-content-center"
+        >
+          <img
+            src="img/AIBrains-logo.png"
+            alt="AI Brains logo"
+            width="300px"
+            height="200px"
+          />
+        </div>
+      </section>
+      <!-- Last Global Summits Section -->
+      <section
+        class="last-summits p-3 p-lg-5 h-100 d-flex flex-column align-items-center gap-3 bg-white"
+      >
+        <h2 class="text-center">See the past AI-Brains Global Summits</h2>
+        <div class="aboutguide-bar m-0 d-flex"></div>
+        <small class="small w-80"
+          >Take a look at the last two AI-B Global Summits which took place in
+          Buenos Aires and in California</small
+        >
+        <div class="images-summit">
+          <div class="images-container">
+            <div
+              class="image1 text-white d-flex flex-column align-items-center justify-content-center fw-bold"
+            >
+              2023<span class="fs-5">AI-B Global Summit 2023 in Lima</span>
+            </div>
+            <div
+              class="image2 text-white d-flex flex-column align-items-center justify-content-center fw-bold"
+            >
+              2021<span class="fs-5"
+                >AI-B Global Summit 2021 in California</span
+              >
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- Partners section -->
+      <section class="partners-section p-3 p-lg-5 h-100">
+        <h2 class="text-center text-white">Partners</h2>
+        <div class="guide-bar"></div>
+        <ul
+          class="partners d-flex gap-3 flex-wrap fs-4 fw-bold p-3 justify-content-center"
+        >
+          <li class="">OpenAI</li>
+          <li class="">Google</li>
+          <li class="">Microsoft</li>
+          <li class="">IBM</li>
+          <li class="">NVIDIA</li>
+        </ul>
+      </section>
+    </main>
+    <!-- Footer section -->
+    <footer>
+      <section
+        class="footer-section d-flex gap-5 p-4 fs-6 justify-content-center align-items-center bg-white"
+      >
+        <a href="index.html"
+          ><img src="img/AIBrain.ico" alt="AI brain logo" width="30px" /> AI
+          <span>B</span>rains</a
+        >
+        <div class="d-flex flex-column text-center">
+          <small class="fw-bold">2023 AIBrains Corp.</small>
+          <small class="fw-bold">All rights reserved</small>
+        </div>
+      </section>
+    </footer>
+    <script src="scripts/script.js"></script>
```

Comment:
- [ ] Great job for creating the footer in the about page in the desktop version  but I highly recommend to remove the partner section and to keep the footer with changing the background color to dark color as shown in the design so please fix it ✅🚀🔥

- your 
![image](https://user-images.githubusercontent.com/58553711/235039882-44d570ff-3e41-43e6-a62f-56927294776c.png)


- Given
![image](https://user-images.githubusercontent.com/58553711/235039833-5f2218cc-329a-44e6-abf9-509d4736438a.png)